packages/core/src/Stores/notification-store.js

```diff
@@ -182,6 +182,7 @@ export default class NotificationStore extends BaseStore {
             loginid,
             obj_total_balance,
             website_status,
+            is_uk,
```

Comment:
We're sorting these variables alphabetically. Could you help to move this to the proper spot?